### DISCUSSION TITLE
Is it possible to get screen resolution in mobile app?

### DISCUSSION THREAD

**Discussion by gorazspajzer** [AvaloniaUI/Avalonia#10499] [General]
I tried everything but without any success. Is it possible to get width and height of Android device and iOS device. I need this because I'm adding controls through my code so I need to know what is the resolution of the screen.

Thanx!

**Comment by maxkatz6** [comment]
No, as far as I can tell, there is no way to get this information from Avalonia API. Other than using top level element size, which might not include all of the screen space.

We have a Window.Screens API, which potentially can be extended to mobile as well.

**Reply by maxkatz6** [reply]
Or, you can use native APIs.
Android: https://stackoverflow.com/questions/4743116/get-screen-width-and-height-in-android
iOS: https://stackoverflow.com/questions/5677716/how-to-get-the-screen-width-and-height-in-ios

Note, you can access native APIs from the net6-ios or net6-android projects.

**Reply by gorazspajzer** [reply]
Thanx, this is what I was looking for! You saved my day!
